### PR TITLE
Admin Create Daily Menu

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: app/DailyMenu.php [hunk 1]
```diff
@@ -14,9 +14,31 @@ class DailyMenu extends Model
...
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = ['quantity', 'created_at', 'updated_at'];
+
+    /**
+     * Get the foods for the menu item.
+     *
+     * @return relationship
```
Comment (ghost) [inline app/DailyMenu.php]: @return Illuminate\Database\Eloquent\Relations\BelongsTo

File: app/Http/Controllers/DailyMenuController.php [hunk 2]
```diff
@@ -13,28 +16,94 @@ class DailyMenuController extends Controller
...
      */
-    public function index()
+    public function create(Request $request)
     {
-        $listDailyMenu = $this->dailyMenu->select('date')->distinct()->orderBy('date', 'desc')->paginate(10);
-        
-        return view('daily_menus.index', ['listDailyMenu' => $listDailyMenu]);
+        $listCategory = $this->category->get();
+        if ($request->ajax()) {
+            $category = $request->category;
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: $category or $category_id?

File: app/Http/Controllers/DailyMenuController.php [hunk 3]
```diff
@@ -13,28 +16,94 @@ class DailyMenuController extends Controller
...
+    {
+        $date = $request['date'];
+        $foodId = $request['food_id'];
+        $quantity = $request['quantity'];
+
+        /**
+         * Check if food has existed in DB, if true then add new quantity with old quantity,
+         * If false then add this food into menu
+         */
+        if (sizeof($this->dailyMenu->where('date', $date)->where('food_id', $foodId))>1) {
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: you should use check exist by querybuilder.

File: app/Http/Controllers/DailyMenuController.php [hunk 4]
```diff
@@ -13,28 +16,94 @@ class DailyMenuController extends Controller
...
+            $oldQuantity = $this->dailyMenu
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: you should use firstOrFail() instead of get()[0].

**Review by ghost — CHANGES_REQUESTED** [review]
check comment.

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much

File: app/Http/Controllers/DailyMenuController.php [hunk 5]
```diff
@@ -13,28 +16,92 @@ class DailyMenuController extends Controller
...
+        $foodId = $request['food_id'];
+        $quantity = $request['quantity'];
+
+        /**
+         * Check if food has existed in DB, if true then add new quantity with old quantity,
+         * If false then add this food into menu
+         */
+        $dailyMenu = $this->dailyMenu->where('date', $date)->where('food_id', $foodId);
+        if ($dailyMenu->count()) {
+            $oldQuantity = $dailyMenu->first()['quantity'];
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: $dailyMenu->first()->quantity

File: app/Http/Controllers/DailyMenuController.php [hunk 6]
```diff
@@ -13,28 +16,92 @@ class DailyMenuController extends Controller
...
+    {
+        $date = $request['date'];
+        $foodId = $request['food_id'];
+        $quantity = $request['quantity'];
+
+        /**
+         * Check if food has existed in DB, if true then add new quantity with old quantity,
+         * If false then add this food into menu
+         */
+        $dailyMenu = $this->dailyMenu->where('date', $date)->where('food_id', $foodId);
```
Comment (at-duyho) [inline app/Http/Controllers/DailyMenuController.php]: you should use firstOrFail()

File: app/Http/Controllers/DailyMenuController.php [hunk 7]
```diff
@@ -13,28 +16,92 @@ class DailyMenuController extends Controller
...
+        if ($dailyMenu->count()) {
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: you should use first() and check null or you use updateOrCreate function.

**Review by ghost — CHANGES_REQUESTED** [review]
Check comment

File: resources/views/daily_menus/create.blade.php [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+				    </div>
+				@endif
+				@if(session()->has('message.level'))
+				    <div class="alert alert-{{ session('message.level') }}"> 
+				    {!! session('message.content') !!}
+				    </div>
+				@endif
+				<div class="form-group">
+                  <label for="dateChooser">{{ _('Choose Date') }}: </label>
+                  @if(!empty($date))
```
Comment (ghost) [inline resources/views/daily_menus/create.blade.php]: why don't you use positive condition?

**Review by ghost — CHANGES_REQUESTED** [review]
check comment.

**Comment by hungbkdn** [comment]
I have updated Ajax request for Food List, please check it again
Thank you so much!

**Review by ghost — APPROVED** [review]
fix conflict

**Review by ghost — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]

**Pull request merged**